modules/widgets/top-posts.php

```diff
@@ -105,6 +130,14 @@ function update( $new_instance, $old_instance ) {
 			$instance['count'] = 10;
 		}
 
+		$allowed_post_types = array_values( get_post_types( array( 'public' => true ) ) );
+		$instance['types'] = $new_instance['types'];
+		foreach( $new_instance['types'] as $type ) {
+			if ( ! in_array( $type, $allowed_post_types ) ) {
+				unset( $type );
```

Comment:
Got it. Thanks for the clarification in Slack! I made some changes in 8de214b2c6bd5ef0a46c1c0090be8acc8ed4d6bd. Tested them, everything seems to work.